lib/dotcom/trip_plan/input_form.ex

```diff
@@ -244,7 +229,9 @@ defmodule Dotcom.TripPlan.InputForm do
       |> selected_modes()
     end
 
-    def selected_modes([]), do: "No transit modes selected"
+    def selected_modes(%{RAIL: true, SUBWAY: true, BUS: true, FERRY: true}), do: "All modes"
+
+    def selected_modes(%{}), do: "Walking directions only"
```

Comment:
```suggestion
```

This doesn't do what it looks like it does. The `"Walking directions only"` chunk would match any map passed in as a parameter, and I'm reasonably sure that the only reason it doesn't short-circuit and catch all inputs is because it never matches because of `def selected_modes(%__MODULE__{} = modes)` a few lines up, and is actually dead code.